chrome/src/manifest.json

```diff
@@ -1,5 +1,5 @@
 {
-  "manifest_version": 2,
+  "manifest_version": 3,
```

Comment:
Migration guide: https://developer.chrome.com/docs/extensions/develop/migrate

Think we also have to move the host permissions below to their own section: https://developer.chrome.com/docs/extensions/develop/migrate/manifest#update-host-permissions

https://github.com/fawind/picasso/blob/43c50ce2cf8bdeecdbe5d3f64740c9a41477cf4a/chrome/src/manifest.json#L7-L10